frontend/test/unit/specs/stores/feature-flag-store.spec.js

```diff
@@ -164,13 +167,13 @@ describe("Feature flag store", () => {
   `(
     "handles fallback for missing $environment",
     ({ environment, flagStatus }) => {
-      const featureFlagStore = useFeatureFlagStore()
-      featureFlagStore.$nuxt.$config.deploymentEnv = environment
-      expect(
-        useFeatureFlagStore().getFlagStatus({
-          status: { staging: "switchable" },
-        })
-      ).toEqual(flagStatus)
+      const initialValue = process.env.DEPLOYMENT_ENV
+      process.env.DEPLOYMENT_ENV = environment
+      const actualStatus = getFlagStatus({
+        status: { staging: "switchable" },
+      })
+      expect(actualStatus).toEqual(flagStatus)
+      process.env.DEPLOYMENT_ENV = initialValue
```

Comment:
Modifying `process.env` needs to be in a `beforeTest` and `afterTest` hook (or whatever the Playwright equivalent is). If `expect` fails and raises, it will never be set back to its initial value.

Reply:
TIL, thank you! I'll update

Reply:
Thank you for explaining the reason!